front/cypress/e2e/admin/pages_and_menu/update_hero_banner_content.cy.ts

```diff
@@ -13,27 +20,51 @@ describe('Admin: update Hero Banner content', () => {
   const updatedSignedInCTAButton = 'signedin button!';
   const updatedSignedInCTAURL = 'https://www.example.biz';
 
-  it('displays hero banner settings on the landing page correctly', () => {
-    // set default homepage settings
-    cy.apiUpdateHomepageSettings({
-      top_info_section_enabled: false,
-      bottom_info_section_enabled: false,
-      events_widget_enabled: false,
-      banner_avatars_enabled: false,
-      banner_layout: 'full_width_banner_layout',
-      banner_signed_out_header_multiloc: {
-        en: signedOutHeaderEnglish,
-      },
-      banner_signed_out_subheader_multiloc: {
-        en: signedOutSubheaderEnglish,
-      },
-      banner_cta_signed_out_text_multiloc: {
-        en: signedOutCTAButton,
-      },
-      banner_signed_out_header_overlay_color: '#33FFD1',
-      banner_signed_out_header_overlay_opacity: 55,
+  const defaultHomepageSettings = {
+    top_info_section_enabled: false,
+    bottom_info_section_enabled: false,
+    events_widget_enabled: false,
+    banner_avatars_enabled: false,
+    banner_layout: 'full_width_banner_layout',
+    banner_signed_out_header_multiloc: {
+      en: signedOutHeaderEnglish,
+    },
+    banner_signed_out_subheader_multiloc: {
+      en: signedOutSubheaderEnglish,
+    },
+    banner_cta_signed_out_text_multiloc: {
+      en: signedOutCTAButton,
+    },
+    banner_signed_out_header_overlay_color: '#33FFD1',
+    banner_signed_out_header_overlay_opacity: 55,
+    banner_cta_signed_out_type: 'sign_up_button',
+    banner_cta_signed_in_type: 'no_button',
+  };
+
+  before(() => {
+    // create a fake verified user
+    cy.apiSignup(
```

Comment:
Why do we need to create this user? Is it worth adding a comment?

Reply:
Don't we need it just for one spec `'views the signed-in content properly as a verified user'`? If that's the case, maybe it's better to send this request right there not to slow down other tests?

Reply:
It's because if you aren't verified, the "verify now" tab pops up, and so by faking the verification on the backend we can skip that part and continue the frontend tests we actually want to test. I will add a comment though, it's not obvious here at all. 

We need the `before` block anyway to set the default homepage settings, so I think it's fine to leave the user creation there as well. I don't think there are many cases where people are running just one test from the suite, and on the CI they all run so the call will take the same amount of time here or in the specific test